README.md

```diff
@@ -17,6 +17,8 @@ Load the files straight into your HTML from the ```dist``` folder:
 <link rel="stylesheet" href="soccer.min.css">
 <script src="soccer.min.js"></script>
 ```
+An example html document is included in the distribution as an example. 
```

Comment:
Do you mean within `/docs`?

Reply:
Yes, a HTML containing the relevant markup and classes.

Reply:
Happy to pick this up btw^

Reply:
So, at the moment within `/docs` there already is an html file which is for https://jrobind.github.io/soccer.js/ (`/docs` is used for GitHub pages). I'm not sure how much additional value adding another file will bring. Maybe creating a UI to interact with the library would be a cool thing to do? Let me know if you'd be happy to take this on!

Reply:
If you remove this comment and also the comment regarding the unnecessary wrapping div below, I can merge this!

Reply:
> So, at the moment within `/docs` there is already an html file which is for https://jrobind.github.io/soccer.js/ (`/docs` is used for GitHub pages). I'm not sure how much additional value adding another file will bring. Maybe creating a UI to interact with the library would be a cool thing to do? Let me know if you'd be happy to take this on!

I actually didn't realise that the HTML within the /docs was interactive lol. Agreed, no need for a further example :)

Reply:
> If you remove this comment and also the comment regarding the unnecessary wrapping div below, I can merge this!

That should be done now (slowly learning how to use GitHub properly!)

Reply:
I'd be happy to take up adding a UI functionality for the app. Sounds like a good first js project to get my teeth stuck into. 

Would you recommend glitch as an IDE?

Reply:
Glitch is cool, totally up to you! I use a text editor called VS Code which is pretty popular with frontend devs.